src/App.tsx

```diff
@@ -1,74 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
```

Comment:
Ensure that the `peopleFromServer` import from './data/people' is correctly defined and used. Verify that this module exports the expected data.